### ISSUE TITLE
Link types 84-88 require vdfs

### ISSUE THREAD

**Issue by samulikyytsonen** [HSLdevcom/helmet-model-system#288]
Car assignment crashes with error  "invalid v-d fct pointer {'nvd': 0, 'jnode': 40713, 'inode': 31400} (523 101)". This link is airport connector with type 88. Types 84-88 should be added to parameters.assignment?

**Comment by zptro** [comment]
So you are testing the `olusanya` branch?

**Comment by samulikyytsonen** [comment]
Yes, olusanya branch. I thought issues are branch-specific, but apparently they are not!
